app/assets/stylesheets/static/static_content.scss

```diff
@@ -5,7 +5,7 @@
   box-sizing: border-box;
   padding: $size-40 34px;
   scroll-behavior: smooth;
-  margin: 0 auto;
+  margin: 50px auto;
```

Comment:
We'll want to apply similar logic in [/app/assets/stylesheets/dashboard/dashboard_content.scss](https://github.com/ifmeorg/ifme/blob/main/app/assets/stylesheets/dashboard/dashboard_content.scss).

Or else this is happening on the signed in pages:

![image](https://user-images.githubusercontent.com/3010728/196046520-3f664db7-0e8c-4d4c-a797-f05b6ca83228.png)